.travis.yml

```diff
@@ -301,7 +301,7 @@ script:
         -t ${TRAVIS_REPO_SLUG}
         -f ${HOME}/docker/Dockerfile.travis
         ${HOME}/docker/
-    && rm -rf ${HOME}/ccache/*
+    && rm -rf ${HOME}/ccache/build_${CC}_${BUILD_TYPE}
```

Comment:
We're happy with the fixes, but for our understanding, why is it more helpful to remove fewer caches?

Reply:
We (@nilsdeppe and I) think there is some conflict going on between caches, so I changed this to only delete the one being used instead of all of them.  We (@nilsdeppe and I) don't think this change should make a difference, but there is no reason to delete all of the caches.  So mainly this change is for consistency with the way other dirs are created and deleted

Reply:
I also doubt it will matter, but consistency is good and it will rule out one possible cause of the failures.